endpointmanager/cmd/main.go

```diff
@@ -1,9 +1,88 @@
 package main
 
 import (
+	"context"
+	"encoding/json"
+	"sync"
+	"time"
+
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/config"
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/endpointmanager"
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/endpointmanager/postgresql"
+	"github.com/onc-healthit/lantern-back-end/lanternmq"
+	"github.com/onc-healthit/lantern-back-end/lanternmq/pkg/accessqueue"
 	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
 )
 
+func failOnError(err error) {
+	if err != nil {
+		log.Fatalf("%s", err)
+	}
+}
+
+// getEndptsAndSend gets the current list of endpoints from the database and sends each one to the given queue
+// it continues to repeat this action every time the given interval period has passed
+func getEnptsAndSend(
+	ctx context.Context,
+	qName string,
+	qInterval int,
+	store endpointmanager.FHIREndpointStore,
+	mq *lanternmq.MessageQueue,
+	channelID *lanternmq.ChannelID) {
+
+	for {
+		listOfEndpoints, err := store.GetAllFHIREndpoints(ctx)
+		failOnError(err)
+
+		for i, endpt := range *listOfEndpoints {
+			if i%10 == 0 {
+				log.Infof("Processed %d/%d messages", i, len(*listOfEndpoints))
+			}
+
+			msgBytes, err := json.Marshal(endpt)
```

Comment:
rather than send the entire endpoint, i'd send just the url as that's the only thing the other services need.